runelite-client/src/main/java/net/runelite/client/plugins/driftnet/DriftNetPlugin.java

```diff
@@ -0,0 +1,223 @@
+{
+	private final int NORTH_NET_ID = 31433;
+	private final int SOUTH_NET_ID = 31434;
+
+	DriftNet northNet = new DriftNet();
+	DriftNet southNet = new DriftNet();
+
+	private NPC interacting = null;
+
+	@Getter
+	private HashMap<NPC, Long> fishes = new HashMap<>();
+
+	@Inject
+	private OverlayManager overlayManager;
+
+	@Inject
+	private DriftNetConfig config;
+
+	@Inject
+	private DriftNetOverlay driftNetOverlay;
+
+	@Inject
+	private InfoBoxOverlay infoBoxOverlay;
+
+
+	@Inject
+	private Client client;
+
+	@Getter
+	private HashSet<NPC> bosses = new HashSet<>();
+
+	@Override
+	protected void startUp() throws Exception
+	{
+		overlayManager.add(driftNetOverlay);
+		overlayManager.add(infoBoxOverlay);
+	}
+
+	@Override
+	protected void shutDown() throws Exception
+	{
+		overlayManager.remove(driftNetOverlay);
+		overlayManager.remove(infoBoxOverlay);
+	}
+
+	@Provides
+	DriftNetConfig provideConfig(ConfigManager configManager)
+	{
+		return configManager.getConfig(DriftNetConfig.class);
+	}
+
+	@Subscribe
+	public void onGameObjectSpawned(GameObjectSpawned event)
+	{
+		GameObject gameObject = event.getGameObject();
+		//ObjectComposition objectComposition = client.getObjectDefinition(gameObject.getId());
+		if (event.getGameObject().getId() == SOUTH_NET_ID)
+		{
+			southNet.setDriftNet(gameObject);
+		}
+		else if (event.getGameObject().getId() == NORTH_NET_ID)
+		{
+			northNet.setDriftNet(gameObject);
+		}
+	}
+
+	@Subscribe
+	public void onMenuEntryAdded(MenuEntryAdded event)
+	{
+		DriftNet net = null;
+		if (event.getIdentifier() == SOUTH_NET_ID)
+		{
+			net = southNet;
+		}
+		else if (event.getIdentifier() == NORTH_NET_ID)
+		{
+			net = northNet;
+		}
+		if (net != null)
+		{
+			if (event.getOption() != null)
+			{
+				if (event.getTarget().toLowerCase().contains("(full)"))
+				{
+					net.setNetStatus(DriftNet.DriftNetStatus.FULL);
+				}
+				else if (event.getOption().toLowerCase().contains("set"))
+				{
+					net.setNetStatus(DriftNet.DriftNetStatus.UNSET);
+				}
+				else
+				{
+					net.setNetStatus(DriftNet.DriftNetStatus.SET);
+				}
+			}
+		}
+	}
+
+	@Subscribe
+	public void onGameObjectDespawned(GameObjectDespawned event)
+	{
+		if (event.getGameObject().getId() == SOUTH_NET_ID)
+		{
+			southNet.setDriftNet(null);
+		}
+		else if (event.getGameObject().getId() == NORTH_NET_ID)
+		{
+			northNet.setDriftNet(null);
+		}
+	}
+
+	@Subscribe
+	public void onNpcSpawned(NpcSpawned event)
+	{
+		NPC npc = event.getNpc();
+		if (npc.getId() == NpcID.FISH_SHOAL)
+		{
+			fishes.put(npc, -1L);
+		}
+	}
+
+	@Subscribe
+	public void onNpcDespawned(NpcDespawned event)
+	{
+		NPC npc = event.getNpc();
+		if (npc.getId() == NpcID.FISH_SHOAL)
+		{
+			fishes.remove(npc);
+		}
+	}
+
+	@Subscribe
+	public void onInteractingChanged(InteractingChanged event)
+	{
+		if (event.getSource() == client.getLocalPlayer())
+		{
+			if (event.getTarget() instanceof NPC)
+			{
+				if (((NPC) event.getTarget()).getId() == NpcID.FISH_SHOAL)
```

Comment:
These if statements should be combined, and displayed over multiple lines. For example:
```java
if (condition1
	&& condition2
	&& condition3)
```